contributing/code/standards.rst

```diff
@@ -89,8 +97,8 @@ short example containing most features described below::
                 'another_default' => 'more values',
             ];
 
-            foreach ($options as $option) {
-                if (!in_array($option, $defaultOptions)) {
+            foreach (array_keys($options) as $option) {
```

Comment:
not sure if you prefer `foreach ($options as $option => $unused) {`

Reply:
`foreach ($options as $name => $values)` would prevent an array copy